src/main/java/com/aws/greengrass/telemetry/SystemMetricsEmitter.java

```diff
@@ -49,34 +115,45 @@ public void emitMetrics() {
         List<Metric> metricsList = new ArrayList<>();
         long timestamp = Instant.now().toEpochMilli();
 
+#if ANDROID
+        double cpuLoad = getCpuLoad();
+        long openFileDescriptorsCount = getOpenFileDescriptorsCount();
+        long usedMemory = getUsedMemory();
+#else
+        double cpuLoad = cpu.getSystemCpuLoadBetweenTicks(previousTicks);
+        previousTicks = cpu.getSystemCpuLoadTicks();
+
+        long openFileDescriptorsCount = systemInfo.getOperatingSystem().getFileSystem().getOpenFileDescriptors();
+
+        GlobalMemory memory = systemInfo.getHardware().getMemory();
+        long usedMemory = memory.getTotal() - memory.getAvailable();
+#endif /* ANDROID */
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, @ expected

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.